sos/plugins/rhv_log_collector_analyzer.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2018 Red Hat, Inc.
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+from sos.plugins import Plugin, RedHatPlugin
+
+
+class RhvLogCollectorAnalyzer(Plugin, RedHatPlugin):
+    """RHV Log Collector Analyzer"""
+
+    packages = ('rhv-log-collector-analyzer',)
+
+    plugin_name = 'rhv_log_collector_analyzer'
+    profiles = ('virt',)
+
+    def setup(self):
+        report = "{dircmd}/rhv-log-collector-analyer-report.html".format(
+            dircmd=self.get_cmd_output_path()
+        )
+
+        self.get_cmd_output_now(
+            "rhv-log-collector-analyzer"
+            " --live"
```

Comment:
Should we gate this on the presence of the ovirt-engine package as well? Are there any situations where `rhv-log-collector-analyzer` might be run on a non-ovirt-manager installation?

Reply:
When using --live requires Engine but it is possible to run with remote Engine.

Reply:
@dankenigsberg yes, I think for now we should add the requirement for now.

Reply:
Thinking twice here, I wouldn't add such requirement static in the plugin because:
- we have such verification in our spec [1]
- rhv-log-collector-analyzer --live soon or later will analyze hosts as well

[1]
'''
%if 0%{?rhel} >= 7 || 0%{?fedora}
Requires:       ovirt-engine
%else
Requires:       rhevm-setup
%endif
''''

Reply:
If it just comes down to using or not using `--live` then just check for the package presence (or some other test) and add or don't add it as appropriate.